demo/api/schema.gql

```diff
@@ -785,6 +852,11 @@ enum ProductTagSortField {
 
 type Mutation {
   currentUserSignOut: String!
+  userPermissionsCreatePermission(data: CreateUserPermissionInput!): UserPermission!
+  userPermissionsUpdatePermission(data: UpdateUserPermissionInput!): UserPermission!
+  userPermissionsSetPermissionContentScopes(data: UserPermissionContentScopesInput!): UserPermission!
+  userPermissionsDeletePermission(id: ID!): Boolean!
+  userPermissionsSetContentScope(userId: String!, contentScopes: [JSONObject!]!): [JSONObject!]!
```

Comment:
Why Set and not Update?

Reply:
Since the operation completely replaces the contentScopes `set` just sounded more logical to me. Do we have guidelines for the naming?

Reply:
https://github.com/vivid-planet/comet/pull/1223/commits/3d41b7594dff6331ee7f9d4c410131e7953b5beb